uiowa/uiowa
# [ITS] helpdesk emails

Resolves https://github.com/uiowa/uiowa/issues/7417

# How to test

```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /node/add/alert
```
- Code review
- Run `ddev launch -m` which will bring up the local email client. 
- Add alert and set "needs review" as status
- Check mailpit and see that an alert email is there
- Test email link in body and review email subject, from address, body
